### PR TITLE
KEYCLOAK-12743 fix: right ignore pattern so that it outputs

### PR DESCRIPTION
**Pull request by edewit**
fixing KEYCLOAK-12743

### CODE REVIEW

**Comment by stianst** [comment]
@edewit If you haven't already read https://github.com/keycloak/keycloak/blob/master/CONTRIBUTING.md, please do. There's some pointers there around expectations in a PR.

One thing is can you please include the JIRA issue in the commit message, like so: "KEYCLOAK-9876 My super cool new feature". It makes it a lot easier to correlate Git commits to JIRAs.

**Comment by abstractj** [comment]
Reassigning it to @ssilvert as he's the best person to handle this.

**Comment by edewit** [comment]
@ssilvert I have `fixing KEYCLOAK-12743` in the commit message

**Comment by edewit** [comment]
changed it to start the commit message with the issue number instead

**Comment by ssilvert** [comment]
> @ssilvert I have `fixing KEYCLOAK-12743` in the commit message

I'm pretty sure it needs to be in the subject line to trigger our tooling.  Fixed it for you.

**Comment by edewit** [comment]
@ssilvert can we merge this one? Without it I can run lint on the command line.

**Review by ssilvert — COMMENTED** [review]
Putting this on hold.  It will be affected by snowpack changes.

File: themes/src/main/resources/theme/keycloak-preview/account/resources/package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "build": "tsc --noImplicitAny --strictNullChecks --jsx react -p ./ && npm run lint",
     "build:watch": "tsc --noImplicitAny --strictNullChecks --jsx react -p ./ -w",
-    "lint": "eslint --ignore-pattern *.d.ts ./**/*.ts*"
+    "lint": "eslint --ignore-pattern **/*.d.ts ./app/**/*.ts*"
```
[the remark this answers is not in the document]
Comment (edewit) [reply themes/src/main/resources/theme/keycloak-preview/account/resources/package.json:8]: it's also linting stuff in node_modules

**Comment by edewit** [comment]
updated to include Main.tsx

**Comment by ssilvert** [comment]
Closing this one as out of date.  The snowpack changes will affect this change, so we will need to just take care of linting after that under KEYCLOAK-1273.

**Pull request closed without merging**